integreat_cms/cms/templates/content_revisions.html

```diff
@@ -6,24 +6,22 @@
     {% load rules %}
     <div class="mb-6">
         <h1 class="heading">
-            {% with page_translations.first.title as page_title %}
-                {% blocktrans %}Page versions of "{{ page_title }}"{% endblocktrans %}
+            {% with content_translations.first.title as content_title %}
+                {% blocktrans %}"Versions of {{ content_title }}"{% endblocktrans %}
             {% endwith %}
         </h1>
-        <a href="{% url 'edit_page' page_id=page.id region_slug=request.region.slug language_slug=language.slug %}"
-           class="btn btn-outline">
-            <i icon-name="arrow-left-circle" class="align-top"></i> {% translate "Back to the page form" %}
+        <a href={{ form }} class="btn btn-outline">
+            <i icon-name="arrow-left-circle" class="align-top"></i> {% translate "Back to the form" %}
         </a>
     </div>
-    <form method="post"
-          action="{% url 'page_revisions' page_id=page.id region_slug=request.region.slug language_slug=language.slug %}">
+    <form method="post" action={{ revision }}>
         {% csrf_token %}
         <div class="w-3/5 m-auto mb-28 relative">
             <input class="relative z-10 pt-1"
                    type="range"
                    name="revision"
                    min="1"
-                   max="{{ page_translations.count }}"
+                   max="{{ content_translations.count }}"
```

Comment:
While we're at it, we could also refactor the existing template and make it more performant - if we call `{{ content_translations.count }}` several times in a template, it will make a new sql query each time. It's better to force the evaluation into a python object and call `{{ content_translations|length }}` each time, which will do the computation in Python instead of SQL and does not retrieve the objects from the database again.